### DISCUSSION TITLE
Cannot login

### DISCUSSION THREAD

**Discussion by abdullaadnan** [getcandy/hub#51] [Help]
Sir, I am using the tag of 0.12.1 hub after the dev command when I try to login my console gives back 500 as error code also sometimes shows invalid credentials but I am sure that I entered valid credential that I put in the installation process. 
I also check #39 its wasn't work for me

![Screenshot from 2021-04-07 14-48-19](https://user-images.githubusercontent.com/47655583/113843763-80275900-97b1-11eb-8ab5-219f85eaff1c.png)

console screenshot:
![Screenshot from 2021-04-07 14-52-45](https://user-images.githubusercontent.com/47655583/113843801-887f9400-97b1-11eb-8992-b677b33a3ddf.png)

**Comment by alecritson** [comment]
Looks like there are some things missing from your install. Have you gone through all the steps outlined here: https://docs.getcandy.io/api/getting-started/installation.html#installations-steps

**Reply by abdullaadnan** [reply]
I tried many times as in that documentation I also used passport instead of sanctum I cannot log in

the console:
[
![Screenshot from 2021-04-07 20-43-18](https://user-images.githubusercontent.com/47655583/113890890-330fab00-97e2-11eb-951b-97738723adce.png)
]

network:
![Screenshot from 2021-04-07 20-43-58](https://user-images.githubusercontent.com/47655583/113890995-4c185c00-97e2-11eb-889e-df7f15cc0330.png)

**Comment by abdullaadnan** [comment]
![Screenshot from 2021-04-08 15-25-48](https://user-images.githubusercontent.com/47655583/114007328-df9f6a80-987e-11eb-9551-9a282a3be554.png)
